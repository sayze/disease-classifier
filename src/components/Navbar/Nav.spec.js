import React from 'react'
import { shallow } from 'enzyme'
import NavBar from '.'

describe('NavBar', () => {
  it('should render default NavBar', () => {
    const component = shallow(<NavBar />)
    expect(component).toMatchSnapshot()
  })
})
