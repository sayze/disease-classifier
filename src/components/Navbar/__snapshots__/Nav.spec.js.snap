// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should render default NavBar 1`] = `
<WithStyles(ForwardRef(AppBar))
  elevation={1}
  position="sticky"
>
  <WithStyles(ForwardRef(Toolbar))
    variant="dense"
  >
    <WithStyles(Logo) />
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
